README.md

```diff
@@ -1,602 +1,361 @@
-# rspec-rails [![Build Status](https://secure.travis-ci.org/rspec/rspec-rails.svg?branch=master)](http://travis-ci.org/rspec/rspec-rails) [![Code Climate](https://img.shields.io/codeclimate/github/rspec/rspec-rails.svg)](https://codeclimate.com/github/rspec/rspec-rails)
-**rspec-rails** is a testing framework for Rails 3.x, 4.x and 5.x.
+# rspec-rails [![Build Status][]][travis-ci] [![Code Climate][]][code-climate]
 
-Use **[rspec-rails 1.x](http://github.com/dchelimsky/rspec-rails)** for Rails
-2.x.
+`rspec-rails` brings the [RSpec][] testing framework to [Ruby on Rails][]
+as a drop-in alternative to its default testing framework, Minitest.
 
-## Installation
+In RSpec, tests are not just scripts that verify your application code;
+they’re also detailed explanations of how the application is supposed to behave,
+expressed in plain English—in other words, _specs._
```

Comment:
Grammatically this bugs me, `English-in other words`, I think `English, or in other words,` might be better.

Reply:
No because `in other words, specs` is not a sentence, therefore a semicolon isn't correct either.

Reply:
You're absolutely right. How would you feel about a colon there instead?

Reply:
I'd prefer a normal comma.

Reply:
Sure thing.